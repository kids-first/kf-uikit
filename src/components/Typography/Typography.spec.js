/* eslint-disable */
import React from 'react';
import { H1, H2 } from '.';
import { createSerializer } from 'jest-emotion';
import * as emotion from 'emotion';
import styled from 'react-emotion';

test('H1 renders with correct styles', () => {
  const actual = create(<H1>hello world</H1>);
  expect(actual).toMatchSnapshot();
});

test('H2 renders with correct styles', () => {
  const actual = create(<H2>hello world</H2>);
  expect(actual).toMatchSnapshot();
});
