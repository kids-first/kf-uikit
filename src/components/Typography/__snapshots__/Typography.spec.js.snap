// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 renders with correct styles 1`] = `
.h1, h1 {
  font-family: Montserrat,Helvetica,sans-serif;
  font-weight: 500;
  font-family: Montserrat,Helvetica,sans-serif;
  -webkit-text-decoration: none;
  text-decoration: none;
  border: none;
  font-size: 36px;
  font-weight: 500;
  -webkit-letter-spacing: 0.5px;
  -moz-letter-spacing: 0.5px;
  -ms-letter-spacing: 0.5px;
  letter-spacing: 0.5px;
  background-image: linear-gradient(to right,#2b388f,#009bb8 51%,#02b0ed), linear-gradient(#2b388f,#2b388f);
  background-clip: text;
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  color: #2b388f;
  font-weight: 500;
}

<h1
  className="h1, h1"
  color="secondary"
  fontWeight="normal"
>
  hello world
</h1>
`;

exports[`H2 renders with correct styles 1`] = `
.h2, h2 {
  font-family: Montserrat,Helvetica,sans-serif;
  font-weight: 500;
  font-family: Montserrat,Helvetica,sans-serif;
  -webkit-text-decoration: none;
  text-decoration: none;
  border: none;
  color: #2b388f;
  font-size: 30px;
  line-height: 0.95;
  -webkit-letter-spacing: 0.5px;
  -moz-letter-spacing: 0.5px;
  -ms-letter-spacing: 0.5px;
  letter-spacing: 0.5px;
  color: #2b388f;
  font-weight: 500;
  color: #2b388f;
  font-weight: 500;
}

<h2
  className="h2, h2"
  color="secondary"
  fontWeight="normal"
>
  hello world
</h2>
`;

exports[`H3 renders with correct styles 1`] = `
.h1, h1 {
  font-family: Montserrat,Helvetica,sans-serif;
  font-weight: 500;
  font-family: Montserrat,Helvetica,sans-serif;
  -webkit-text-decoration: none;
  text-decoration: none;
  border: none;
  font-size: 36px;
  font-weight: 500;
  -webkit-letter-spacing: 0.5px;
  -moz-letter-spacing: 0.5px;
  -ms-letter-spacing: 0.5px;
  letter-spacing: 0.5px;
  background-image: linear-gradient(to right,#2b388f,#009bb8 51%,#02b0ed), linear-gradient(#2b388f,#2b388f);
  background-clip: text;
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  color: #2b388f;
  font-weight: 500;
}

.h3, h3 {
  font-family: Montserrat,Helvetica,sans-serif;
  font-weight: 500;
  font-family: Montserrat,Helvetica,sans-serif;
  -webkit-text-decoration: none;
  text-decoration: none;
  border: none;
  font-weight: 500;
  font-size: 18px;
  line-height: 1.44;
  -webkit-letter-spacing: 0.5px;
  -moz-letter-spacing: 0.5px;
  -ms-letter-spacing: 0.5px;
  letter-spacing: 0.5px;
  color: #2b388f;
  color: #2b388f;
  font-weight: 500;
}

<h3
  className="h3, h3"
  color="secondary"
  fontWeight="normal"
>
  hello world 
  <h1
    className="h1, h1"
    color="secondary"
    fontWeight="normal"
  >
    foobar
  </h1>
</h3>
`;
